pkgs/development/tools/parsing/quickbms/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, fetchurl, pkgconfig, unzip, lzo, bzip2, zlib, openssl }:
+
+with stdenv.lib;
+stdenv.mkDerivation rec {
+  version = "0.10.1";
+  pname = "quickbms";
+
+  src = fetchurl {
+    url = "http://aluigi.zenhax.com/papers/quickbms-src-${version}.zip";
+    sha256 = "172588jc60irrafjwz4dhy0h08hc7s5c68mmq5v6140yv3f9ixc0";
+  };
+
+  nativeBuildInputs = [ unzip ];
+  buildInputs = [ lzo bzip2 zlib openssl ];
+
+  makeFlags = [ "PREFIX=$(out)" ];
+
+  meta = {
+    description = "Files extractor and reimporter, archives and file formats parser";
+    homepage = "http://quickbms.com";
+    license = licenses.gpl2;
```

Comment:
The gpl license variants without the `Plus` or `Only` suffix are deprecated, because they do not specify the license well enough. Use `Plus` if the project has "or later" sections in the actual source code files (not the license). Otherwise, use the `Only` suffix.
 
See https://discourse.nixos.org/t/lib-licenses-gpl3-co-are-now-deprecated/8206 for more details.